FunctionalTests/Microsoft.Bot.Builder.FunctionalTests/WebexClientTest.cs

```diff
@@ -47,43 +48,49 @@ public async Task SendAndReceiveWebexMessageShouldSucceed()
 
         private async Task SendMessageAsync(string echoGuid)
         {
-            var data = new NameValueCollection
+            var data = new FormUrlEncodedContent(new[]
             {
-                ["roomId"] = _roomId,
-                ["text"] = echoGuid,
-            };
+                new KeyValuePair<string, string>("roomId", _roomId),
+                new KeyValuePair<string, string>("text", echoGuid)
+            });
 
-            using (var client = new WebClient())
-            {
-                client.Headers[HttpRequestHeader.Authorization] = "Bearer " + _userAccessToken;
-                await client.UploadValuesTaskAsync($"{WebexUrlBase}/messages", "POST", data);
-            }
+            using var client = new HttpClient();
+
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _userAccessToken);
+
+            await client.PostAsync($"{WebexUrlBase}/messages", data);
         }
 
         private async Task<string> ReceiveMessageAsync()
         {
-            using (var client = new WebClient())
+            var resultMessage = string.Empty;
```

Comment:
resultMessage is never assigned. Why not get rid of it and just return string.Empty on line 93?

Reply:
That's true, fixed. Thanks!